### DISCUSSION TITLE
[Feature request] changing chat Fonts Option

### DISCUSSION THREAD

**Discussion by carnifexx** [baffler/Transparent-Twitch-Chat-Overlay#61] [Ideas]
Probably an obvious one and I know it might be hard work and a bit too much to ask for.

Still, it would be great to change the color of chatter names and messages and choose fonts and font size. Also changing Emoji Sizes would be great.

Thank you so much for the great work though, In OBS just "**window capture**" Transparent-Twitch-Chat-Overlay with capture method "**Windows Graphics Capture (windows 10 10 1903 and up)**" put the **Luma Key** > "Luma Max 1,00" filter on it, and bang, transparent chat overlay without any other tool or service, its great.

**Comment by DasMonschta** [comment]
@baffler Thank you for implementing this font size option. I could not use any other overlays, because i need bigger font size and the other tools cant change font size. Thank you.
